cpanfile

```diff
@@ -1,4 +1,4 @@
-requires 'Archive::Extract';
+requires 'Archive::Extract', '> 0.7';
```

Comment:
This is to remove the warning that the Archive::Extract is going to be removed from perl core.

This warning, makes some tests failing.

Reply:
It's even packaged already! :D